types/video.js/index.d.ts

```diff
@@ -6195,6 +6207,35 @@ export interface VideoJsPlayer extends videojs.Component {
 
     aspectRatio(): string;
 
+    /**
+     * Get the current audioOnlyMode state or set audioOnlyMode to true or false.
+     *
+     * Setting this to `true` will hide all player components except the control bar,
+     * as well as control bar components needed only for video.
+     *
+     * @param [value]
+     *         The value to set audioOnlyMode to.
+     *
+     * @return A Promise is returned when setting the state, and a boolean when getting
+     *         the present state
+     */
+    audioOnlyMode(value: boolean): void;
+
+    audioOnlyMode(): Promise<void> | boolean;
+
+    /**
+     * Get the current audioPosterMode state or set audioPosterMode to true or false
+     *
+     * @param [value]
+     *         The value to set audioPosterMode to.
+     *
+     * @return A Promise is returned when setting the state, and a boolean when getting
+     *         the present state
+     */
+    audioPosterMode(value: boolean): void;
+
+    audioPosterMode(): Promise<void> | boolean;
```

Comment:
```suggestion
    audioPosterMode(value: boolean): Promise<void>
    audioPosterMode(): boolean;
```
https://github.com/videojs/video.js/blob/3ec2ac7f99704434d2fdc61642247ef0aa4e77b3/test/unit/player.test.js#L1608-L1612